src/charm.py

```diff
@@ -445,7 +443,7 @@ def _make_pebble_config(self, env_config) -> dict:
                 CONTAINER_NAME: {
                     "override": "replace",
                     "summary": "content-cache",
-                    "command": "/usr/sbin/nginx -g 'daemon off;'",
+                    "command": "/srv/content-cache/entrypoint.sh",
```

Comment:
As per previous comment, I think we should avoid an entrpoint script and manage what the script does in the charm code

Reply:
Sounds good, I'll take a look and try to reorganize things if possible

Reply:
It looks like itwon't be possible to get rid of the entrypoint, since we need it as a workaround for https://bugs.launchpad.net/juju/+bug/1894782